static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -82,16 +82,24 @@ define([
         return messageShown;
     }
 
+    function isInTest(testId, variant) {
+        var participations = storage.local.get('gu.ab.participations');
```

Comment:
I don't think you have check local storage yourself. You should just be able to do something like this:
`ab.getTestVariantId('nameOfTest') &&  ab.testCanBeRun('nameOfTest') `